packages/suite/src/views/wallet/anonymize/components/CoinjoinSetupStrategies.tsx

```diff
@@ -94,10 +99,30 @@ export const CoinjoinSetupStrategies = ({ account }: CoinjoinSetupStrategiesProp
         );
     }
 
-    const maxRounds = getMaxRounds(targetAnonymity, account.addresses.anonymitySet);
+    const maxRounds = getMaxRounds(targetAnonymity, anonymitySet);
     const isCustom = strategy === 'custom';
     const allChecked = connectedConfirmed && termsConfirmed;
-    const fee = new BigNumber(notAnonymized).times(coordinatorData.coordinatorFeeRate).toString();
+    const feeInducingUtxos =
```

Comment:
nit: those calculations could be separate helpers outside of the component to make it more readable :)

Reply:
Moved to `coinjoinUtils`, 51c768b46059ccd7eb060f3f50a7dda5ee935c0b